flink-runtime/src/test/java/org/apache/flink/runtime/rest/messages/ThreadDumpInfoTest.java

```diff
@@ -86,4 +94,12 @@ public void testStacktraceDepthLimitation() {
     private long getOutputDepth(String stringifiedInfo) {
         return Arrays.stream(stringifiedInfo.split("\n")).filter(x -> x.contains("\tat ")).count();
     }
+
+    private String[] removeFirstIndex(String[] lines) {
+        String[] remained = new String[lines.length - 1];
```

Comment:
Could directly use `Arrays.copyOfRange(lines, 1, lines.length);`?

Reply:
Of course! I forgot this method. Thanks for pointing it.